src/pytsql/grammar/sa_tsql.py

```diff
@@ -1,6 +1,6 @@
 # This file was auto-generated by speedy-antlr-tool v1.3.1
 #  https://github.com/amykyta3/speedy-antlr-tool
-
+import logging
```

Comment:
Why `logging` and not using [`warning`](https://docs.python.org/3/library/warnings.html) instead?

Reply:
Good idea